packages/message-manager/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.0.3]
+
+### Changed
+
+- Bump `jsonschema` from `^1.2.4` to `^1.4.1` ([#4998](https://github.com/MetaMask/core/pull/4998), [#5027](https://github.com/MetaMask/core/pull/5027))
```

Comment:
`nth-sig-util` was updated to v8.1.0 in this first PR, but it was reverted in the second. Taken together, only `jsonschema` was updated.